lib/modules/ai_bot/entry_point.rb

```diff
@@ -75,6 +91,12 @@ def inject_into(plugin)
 
         plugin.register_svg_icon("robot")
 
+        plugin.add_to_serializer(
+          :topic_view,
+          :persona_name,
```

Comment:
IMO would be better to keep consistent with `ai_persona`...then at least there is indication that this is added by the AI plugin

Reply:
will do